tests/Gelf/Test/Encoder/JsonEncoderTest.php

```diff
@@ -46,7 +46,7 @@ public function testEncode()
 
         // check that there is JSON inside
         $data = json_decode($json, $assoc = true);
-        $this->assertInternalType('array', $data);
+        $this->assertIsArray($data);
```

Comment:
Does this really work with PHPUnit 5.x and 6.x (as these versions are possible according to composer.json)

Reply:
I don't know, I think we have to release major version for this lib anyway. Since I don't have time for extended changes right now, I'll probably just drop support for PHP <7.4 or even < 8.0 and update dependencies and syntax accordingly.